XO.py

```diff
@@ -0,0 +1,157 @@
+from random import randint
+from collections import namedtuple
+
+# Size of board.
+N = 3
+
+
+class Model:
+    def __init__(self):
+        self.board = [[None for x in range(N)] for y in range(N)]
+
+    def get_board_status(self):
+        return self.board
+
+    def update_board(self, line, col, players, curr_player):
+        if self.board[line][col] is not None:
+            return False
+        else:
+            self.board[line][col] = players[curr_player].symbol
+            return True
+
+    def line_won(self, players, curr_player):

```

Comment:
Very nice implementation of all three winning methods!

Reply:
Thanks :)